### DISCUSSION TITLE
How to set strength of breakable object?

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#103] [Help / Questions] [answered]
Is there a parameter to set the strength of a breakable object? So I could make it weaker or stronger? Thanks!

**Comment by yandeu** [comment, marked as answer]
It is not yet possible to adjust the strength.

Check: https://github.com/enable3d/enable3d/issues/104

**Reply by N8python** [reply]
Awesome, thanks for opening an issue!

**Reply by yandeu** [reply]
You're welcome 🙂
